frame/multisig/src/lib.rs

```diff
@@ -603,6 +604,8 @@ impl<T: Config> Pallet<T> {
 			// Just start the operation by recording it in storage.
 			let deposit = T::DepositBase::get() + T::DepositFactor::get() * threshold.into();
 
+			T::Currency::reserve(&who, deposit)?;
```

Comment:
only logical change of this PR, which seems correct to me.

Reply:
agreed.